src/web/screens/sharingPermissions.tsx

```diff
@@ -133,7 +142,14 @@ function SharingPermissions() {
                     decrypt your UID2 tokens.
                     <br />
                     <br />
-                    Note: This only enables the sharing permission. No data is sent.
```

Comment:
I think we always want to display this note - it's telling participants that enabling sharing with another participant doesn't send any data, it just enables the sharing feature with that participant.

Reply:
Yeah, removing this was a mistake.  Thanks for spotting it.